exercises/059_integers.zig

```diff
@@ -6,7 +6,8 @@
 //     const a2: u8 = 0x41;      // hexadecimal
 //     const a3: u8 = 0o101;     // octal
 //     const a4: u8 = 0b1000001; // binary
-//     const a5: u8 = 'A';       // UTF-8 code point literal
+//     const a5: u8 = 'A';       // ASCII code point literal
+//     const a6: u16 = '⚡'      // Unicode code points can take up to 21 bits
```

Comment:
There is a missig space before the comment.

Reply:
Thanks: my monospace font took more space for this glyph :thinking: . I have chosen another character in the amended commit and also added the missing ";".